.github/workflows/black.yml

```diff
@@ -1,16 +1,13 @@
 name: Black Linter
 
 on:
```

Comment:
For my own understanding: this now means that it only gets run when called from the `isort` action? That seems like a good choice, just wanted to make sure I understand. 

There's a couple things like this that would be confusing at first glance. It looks like you can add comments like in any other yml file with the `#`, so I think adding a few would be helpful. No need to go overboard though if you don't think they're actually useful, the `name` field` does provide some documentation.

Reply:
Yup that is the correct understanding. I'll add some comments to make it more clear.